src/test/java/core/basesyntax/strategy/impl/BalanceHandlerTest.java

```diff
@@ -0,0 +1,44 @@
+package core.basesyntax.strategy.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.OperationHandler;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class BalanceHandlerTest {
+
+    private static OperationHandler balanceHandler;
+    private static FruitTransaction fruitTransaction;
+
+    @BeforeEach
+    void setUp() {
+        balanceHandler = new BalanceHandler();
+        fruitTransaction = new FruitTransaction(FruitTransaction.Operation.BALANCE,"Apple", 10);
```

Comment:
Move this lines to the method beforeAll(), as I suggested in previous comments.
Fix all similar cases in all test classes where needed.